test/unit/test_hpe3par_snapshot.py

```diff
@@ -230,7 +249,10 @@ def test_main_exit_modify(self, mock_modify_snapshot, mock_module, mock_client):
             'allow_remote_copy_parent': None,
             'new_name': 'new_snapshot',
             'rm_exp_time': True,
-            'state': 'modify'
+            'state': 'modify',
```

Comment:
shouldn't it be modify_schedule?

Reply:
No, 'modify_schedule' state is already there, 'modify' stands for snapshot modification